schema/MetaCampaign.sql

```diff
@@ -20,6 +20,8 @@ CREATE TABLE `MetaCampaign` (
   `isEmergencyIMF` tinyint(1) NOT NULL,
   `salesforceLastPull` datetime DEFAULT NULL,
   `salesforceId` varchar(18) DEFAULT NULL,
+  `total_adjustment_amountInPence` int NOT NULL,
+  `total_adjustment_currency` varchar(255) NOT NULL,
```

Comment:
I missed this when we first used the `Money` embeddable, but what do you think about giving it `length: 3`?

Reply:
Yep good idea, done in 9aed4cd8